test/write.spec.ts

```diff
@@ -885,6 +886,7 @@ describe('writeBenchmark()', function () {
             externalDataJsonPath: undefined,
             maxItemsInChart: null,
             failThreshold: 2.0,
+            commitSha: 'dummy sha',
```

Comment:
I think we should add specific tests for the use-case where we set the explicit `commitSha` rather than just adding this param here.